tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -151,6 +153,13 @@ def download_error_report(self, request, pk=None):
         serializer = ParsingErrorSerializer(parser_errors, many=True, context=self.get_serializer_context())
         return Response(get_xls_serialized_file(serializer.data))
 
+    @action(methods=["post"], detail=False, parser_classes=[JSONParser, MultiPartParser, FormParser])
+    def run_action_reparse_cmd(self, request, pk=None):
```

Comment:
this is the command which jquery is calling to run reparse, so task should be assigned to here

Reply:
This can be deleted since it isn't used. If im not mistaken this allows the frontend to make a post request to reparse files which I don't think is what we want at the moment.